test/test_ops.py

```diff
@@ -1702,6 +1705,16 @@ def test_refs_are_in_python_ref_db(self, op):
             return
         self.assertIn(op, self.ref_db_names_set)
 
+    # like 'test_refs_are_in_python_ref_db' but check in the other direction,
+    # this includes aliases since those are not allowed in 'ref_db_names'
```

Comment:
Given that it's so similar, this test should probably be merged with the previous one.

Reply:
these cannot be merged if we want to keep using the decorator. two different sets of ops are used to generate these tests

Reply:
you can just run the second part of the test if it the op is in `ref_alias`.

Reply:
i'm still not getting it. the main point here is to detect a mismatch _between lists of different sizes_, the aliases part is secondary (it's there to make the tests work properly). eg, we could have a ref defined somewhere in the code without a PythonRefInfo entry or vice versa.

simplifying (to not consider matching items and ignoring the aliases), we need to consider these cases:
```
ops_names = [a], db_names = []  => ref_db test fails, but ops_names test passes
ops_names = [],  db_names = [a] => ops_names test fails, but ref_db test passes
ops_names = [a], db_names = [b] => both tests fail
ops_names = [],  db_names = []  => both tests succeed
```

we cannot have only one of these tests because we'll miss out on test coverage (see the first two cases above)

Reply:
ah, nvm, I misread the ops these tests were running over.